src/webob/acceptparse.py

```diff
@@ -408,6 +408,24 @@ def generator(value):
                 )
         return generator(value=value)
 
+    def _parse_and_normalize_offers(self, offers):
+        """
+        Throw out any offers that do not match the media type ABNF.
+
+        :return: A list of ABNF-validated offers split into the format:
+                 ``[offer_index, offer_type_subtype, offer_media_params]``.
```

Comment:
I think `offer_media_params` should be `offer_media_type_params`?

Reply:
fixed